source/jormungandr/jormungandr/street_network/asgard.py

```diff
@@ -93,6 +93,12 @@ def status(self):
             'zmq_socket_ttl': self.socket_ttl,
         }
 
+    def make_location(self, obj):
+        coord = get_pt_object_coord(obj)
+        return type_pb2.LocationContext(
+            place="", access_duration=0, lon=round(coord.lon, 6), lat=round(coord.lat, 6)
+        )
+
```

Comment:
I don't see any use of this function ?

Reply:
https://github.com/CanalTP/navitia/pull/3325/files#diff-f8c05bdbcb56f383bd696ed498742be0932a5b7971a554f4b60bebb44baf238eR179